src/engine/server/server.cpp

```diff
@@ -1915,5 +1902,5 @@ void CServer::SendInitialData(int ClientID)
 
 bool CServer::GetClientProtocolCompatibility(int ClientID, int Protocol) const
 {
-	return (m_aClients[ClientID].m_Protocol == MODAPI_CLIENTPROTOCOL_TW07MODAPI);
+	return (m_aClients[ClientID].m_Protocol == Protocol);
```

Comment:
It's useless now, but if we have several version of ModAPI, the parameter will he usefull. Check message.cpp, I use it.